src/registrar/templates/application_intro.html

```diff
@@ -10,11 +10,11 @@
 
         <h1>You're about to start your .gov domain request.</h1>
```

Comment:
@michelle-rago can we change all instances of ' to ‘ ?

Reply:
ok, done.